### ISSUE TITLE
Some forms are .txt only and web UI wants .html?

### ISSUE THREAD

**Issue by n2ygk** [la5nta/pat#325]
When I try to compose via http interface the template `MAPPPING-GIS FORMS/Local Weather Report` I get a not found error for `http://127.0.0.1:8080/api/forms?formPath=MAPPING-GIS%20FORMS%2FLocal%20Weather%20Report.html`.

Looking in the templates directory, I see that these forms are only available as .txt files. I'm not sure if the missing HTML files are a pat or a Winlink issue (since there's a perfectly fine HTML local weather form in `WEATHER FORMS`).

(Pat is really nice! Finally some amateur UI software that doesn't require Windows.)

**Comment by martinhpedersen** [comment]
I guess we only support HTML forms now? (@xylo04)

If so, I think we should mark those forma as unsupported in the UI or at least give a better error message.

Let's call this a bug 🙂

**Comment by n2ygk** [comment]
Or
```
(echo '<HTML><PRE>'; cat file.txt; echo '</PRE></HTML>)'
```
😃

**Comment by martinhpedersen** [comment]
Would that work? 😁

**Comment by martinhpedersen** [comment]
... if so, we can easily add a fallback wrapping txt-only forms when serving them to the web client 🙂

**Comment by martinhpedersen** [comment]
.. the details are staring to come back to me. IIRC, the html contains logic to accept user input. So we would need to generate that as well.

**Comment by n2ygk** [comment]
Yes HTML forms include user input but I would argue that for text-only
templates it’s not your problem to figure out what the input should be. I
don’t know if these text-only forms are even used by anyone. More of an
upstream problem for winlink to finish the handful of non-HTML templates or
retire them.

**Comment by xylo04** [comment]
I'll add, myself and at least a few other users are following [Winlink Thursday](https://emcomm-training.org/Winlink_Thursdays.html) exercises, and this week's challenge was to download a text-only form and put it into WLE's Private Templates. Well, this issue is preventing use of text only forms, plus Pat doesn't really have a Private Templates directory yet. So this was a reminder to keep widening our feature set! :+1:

**Comment by drboone** [comment]
The FEMA R5AN net this week wants 'Winlink Check In', which is .txt only.
